tests/DynamoDbCompositeModelTest.php

```diff
@@ -150,6 +150,38 @@ public function testStaticMethods()
         ])->toArray());
     }
 
+    public function testConditionContainingCompositeIndexKey()
+    {
+        $fooItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Foo'],
+            'count' => ['N' => 11],
+        ]);
+
+        $barItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Bar'],
+            'count' => ['N' => 9],
+        ]);
+
+        $bazItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Baz'],
+            'count' => ['N' => 10],
+        ]);
+
+        $foundItems = $this->testModel
+            ->where('id', 'id1')
+            ->where('count', '>=', 10) // Test range key support comparison operator other than EQ
```

Comment:
Should we test if the conditions don't have all the composite keys, it will not allow `>=`, etc.?